react/features/overlay/components/FilmStripOnlyOverlayFrame.js

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import {
+    Avatar,
+    getAvatarURL,
+    getLocalParticipant
+} from '../../base/participants';
+
+import OverlayFrame from './OverlayFrame';
+
+/**
+ * Implements a React Component for the frame of the overlays in filmstrip only
+ * mode.
+ */
+class FilmStripOnlyOverlayFrame extends Component {
+    /**
+     * FilmStripOnlyOverlayFrame component's property types.
+     *
+     * @static
+     */
+    static propTypes = {
+        /**
+         * The source (e.g. URI, URL) of the avatar image of the local
+         * participant.
+         *
+         * @private
+         */
+        _avatar: React.PropTypes.string,
```

Comment:
The underscore at the beginning of the name says: non-public. For a Component prop that means: do not pass a value to that property in JSX because this property is not public but you see it because JavaScript does not have built in access control. If `_avatar` is meant to be used that way, then it should have an underscore. If it may be passed a value in JSX, then it's public and, consequently, it shouldn't have an underscore. I haven't looked at the source code of this `_avatar`'s usage.